server/src/main/java/com/lunchtime/security/JwtUtil.java

```diff
@@ -0,0 +1,58 @@
+package com.lunchtime.security;
+
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
+import io.jsonwebtoken.SignatureAlgorithm;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.stereotype.Service;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+
+@Service
+public class JwtUtil {
+
+    private String SECRET_KEY = "lunch_time";
```

Comment:
You shouldn't store secret key in opensource github repository. At least during development time you should save it in application.property

Reply:
allready done